spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -242,18 +251,31 @@ import org.springframework.cloud.gcp.data.datastore.core.mapping.Embedded;
 import org.springframework.data.annotation.Id;
 
 public class EmbeddableTreeNode {
-	@Id
-	long value;
+  @Id
+  long value;
 
-	@Embedded
-	List<EmbeddableTreeNode> children;
+  @EmbeddedComponents
```

Comment:
rename this to `ContainsEmbeddedEntities`

Reply:
no longer necessary since i am now using the entity annotation on the embedded object's class.